### PR TITLE
Fixed Issues raised by PEP8Bear

### LINKED ISSUE #200: PEP8Bear raises error in build
The newly added bear `PEP8Bear` raisies issue in some implementations, this issue is for fixing them

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #200

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous timespan of 7 days.

### CODE REVIEW

File: stack/stack.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+
```
Comment (aviaryan) [inline stack/stack.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

Comment (aviaryan) [inline stack/stack.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

File: quicksort/quick_sort.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def main():
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
+
```
Comment (aviaryan) [inline quicksort/quick_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

Comment (aviaryan) [inline quicksort/quick_sort.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

File: stack/stack.py (hunk 1 above)
Comment (aviaryan) [inline stack/stack.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

File: quicksort/quick_sort.py (hunk 2 above)
Comment (aviaryan) [inline quicksort/quick_sort.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

Comment (aviaryan) [inline quicksort/quick_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

File: stack/stack.py (hunk 1 above)
Comment (aviaryan) [inline stack/stack.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

Comment (aviaryan) [inline stack/stack.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```

File: quicksort/quick_sort.py (hunk 2 above)
Comment (aviaryan) [inline quicksort/quick_sort.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

Comment (aviaryan) [inline quicksort/quick_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```

File: quicksort/quick_sort.py [hunk 3]
```diff
@@ -63,5 +63,6 @@ def main():
     quick_sort(a, 0, len(a) - 1)
     print(a)
 
+
```
Comment (Monal5031) [inline quicksort/quick_sort.py]: @singhpratyush @aviaryan sir gitmate is showing false error here.

Comment (Monal5031) [inline quicksort/quick_sort.py]: Got it sorted 😃

**Pull request merged**